awxkit/awxkit/api/pages/api.py

```diff
@@ -213,11 +213,23 @@ def _export_list(self, endpoint):
         assets = (self._export(asset, post_fields) for asset in endpoint.results)
         return [asset for asset in assets if asset is not None]
 
+    def _check_for_int(self, value):
+        return isinstance(value, int) or (isinstance(value, str) and value.isdecimal())
+
     def _filtered_list(self, endpoint, value):
-        if isinstance(value, int) or value.isdecimal():
+        if isinstance(value, list) and len(value) == 1:
+            value = value[0]
+        if self._check_for_int(value):
             return endpoint.get(id=int(value))
+
         options = self._cache.get_options(endpoint)
         identifier = next(field for field in options['search_fields'] if field in ('name', 'username', 'hostname'))
+        if isinstance(value, list):
+            if all(self._check_for_int(item) for item in value):
+                identifier = 'or__id'
+            else:
+                identifier = 'or__' + identifier
```

Comment:
I used `or__` here as `__in` no longer worked as `requests` is building query params with a list like that:
```
https://url/api/v2/projects/?name__in=name1&name__in=name2
```
and the result was empty. `or__` works better here I think